understanding/reading-level.html

```diff
@@ -58,189 +56,121 @@
                   
                </p>
-               <p>The education level required to read text content (also called "readability") is measured
-                  by analyzing selected passages of text from the Web page. If the Web page includes
-                  text written for different purposes or different styles are used, the selected passages
-                  include samples of the types of content in the Web page and the different styles in
-                  which the content is written. (In many cases, the Web page contains only one kind
-                  of text content—e.g., technical documentation, a legal notice, marketing material,
-                  etc.—and all the content uses the same style.) 
+               <p>テキストのコンテンツを読むのに必要な教育レベル (リーダビリティ (readability) とも呼ばれる) は、ウェブページからテキストの選択された部分を解析することで測定される。ウェブページに異なる目的で書かれた又は異なるスタイルを用いたテキストが含まれている場合、選択された部分は、ウェブページにあるコンテンツの種類及びそのコンテンツが書かれている異なるスタイルのサンプルを含める。(多くの場合、ウェブページには、例えば、技術的な文書、法律上の表示、マーケティング素材など、一つの種類のテキストコンテンツのみが含まれ、そして、すべてのコンテンツは同じスタイルを用いている)。
                   
                </p>
-               <p>Educators can also measure the education level required to read text content. For
-                  example, qualified teachers can evaluate text according to local education standards
-                  to determine if it requires reading ability beyond what is expected for students in
-                  the last year of lower secondary education.
+               <p>教育者は、テキストコンテンツを読むのに必要とされる教育レベルを測定することもできる。例えば、資格のある教師は、前期中等教育の最終学年の生徒に要求される以上の読解力を必要とするかどうかを判断するために、ローカルの教育標準に従って文章を評価することができる。
                   
                </p>
-               <p>Because people's names, the names of cities or other proper names cannot be changed
-                  to shorter names with fewer syllables, and because doing so or just referring to everyone
-                  by pronouns can make sentences harder to understand, the success criterion specifies
-                  that proper names can be ignored or removed from the text before assessing whether
-                  it meets the reading ability requirement. Titles refer to the name of documents, books,
-                  movies, etc. Titles are removed or ignored for the analysis because changing the words
-                  in titles might make the titles easier to read but would make it impossible to understand
-                  the item to which the title refers. This would make it harder to read and understand
-                  the content. (e.g., a book, academic paper, article, etc.). Therefore, titles are
-                  also exempted specifically. 
+               <p>人名、都市名、又はその他の固有名詞は、短い音節で短い名前に変更することはできず、そして、そうすることや代名詞によって全てを参照することは文章を理解しづらくする恐れがあるため、この達成基準は、読解力の要件を満たしているかどうかを検証する前に、固有名詞をテキストから無視又は取り除くことができると述べられている。タイトルは、文書、書籍、映画などの名前を指している。タイトルにある文言を変更すると、読みやすくなるが、タイトルが参照しているアイテムを理解できなくなる可能性があるため、分析する際にタイトルを削除するか無視される。これにより、コンテンツを読み理解することが難しくなる。(例えば、書籍、学術論文、記事など)。したがって、タイトルも明確に免除される。
```

Comment:
ここ結構怪しいですね。「これにより、コンテンツを読み理解することが難しくなる。」という「これ」が何を指すのか非常にわかりにくいです。これ原文と語順が変わっているからだと思います。
あと、達成基準の訳の中ではtitleを「題名」としているので揃えた方がいいかも。
そのほかいろいろ調整してこんな感じでどうでしょうか。

```suggestion
               <p>人名、都市名、又はその他の固有名詞は、短い音節で短い名前に変更することはできず、そして、そうすることや代名詞によって全てを参照することは文章を理解しづらくする恐れがあるため、この達成基準は、読解力の要件を満たしているかどうかを検証する前に、固有名詞をテキストから無視又は取り除いて良いとしている。題名とは、文書、書籍、映画などの名前を指すものである。検証の際に題名を削除するか無視するのは、題名の文言を変更すると読みやすくはなるが、どの項目を参照しているのか理解できなくなる可能性があるためである。これにより、コンテンツを読み理解することが難しくなる。(例えば、書籍、学術論文、記事など)。したがって、タイトルも明確に免除される。
```